### PR TITLE
bug 1341770: Set Yes/No for "is_staff" GA dim

### PR DESCRIPTION
**Pull request by jwhitlock**
The "yesno" template filter also localizes the text, so visits on French pages get the values "Oui" / "Non". Use regular template logic so the values are only "Yes" / "No".  Found during manual testing of #4175.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
LOL, unexpected side effect.

I've thought of some feedback I probably should have had on the first pull request - better late than never?

Rather than saying yes or no, could you put the entire dimension behind the if statement and only set it (to yes) if the user is staff? I think that will make the dimension stick if the staff member is logged out. Otherwise it'll be overridden.

**Comment by jwhitlock** [comment]
Making sure I understand.

Here's how it works today:

* User is anonymous / logged out - dimension18 is not set
* User is logged in, non-staff - dimension18 is set to ``No``
* User is logged in, staff - dimension18 is set to ``Yes``

and you want it changed to:

* User is anonymous / logged out - dimension18 is not set
* User is logged in, non-staff - **dimension18 is not set**
* User is logged in, staff - dimension18 is set to ``Yes``

Is this true?

**Comment by jwhitlock** [comment]
Confirmed w/ @stephaniehobson on IRC, changed behavior to only set dim18 if staff

File: jinja2/includes/google_analytics.html [hunk 1]
```diff
@@ -19,8 +19,10 @@
             ga('set', 'dimension2', 'Yes');
         {% endif %}
 
-        // dimension18 == "Staff"
-        ga('set', 'dimension18', '{{ request.user.is_staff | yesno }}');
+        {%- if request.user.is_staff %}
+            // dimension18 == "Staff"
+            ga('set', 'dimension18', 'Yes');
```
Comment (safwanrahman) [inline jinja2/includes/google_analytics.html]: So If ``resuest.user`` is not ``superuser`` it should not send data to GA?

Comment (stephaniehobson) [inline jinja2/includes/google_analytics.html]: Correct.

**Comment by stephaniehobson** [comment]
👍

**Pull request merged**